.gitignore

```diff
@@ -27,6 +28,7 @@
 /config/application.yml
 
 /spec/fixtures/temp_schools_data.csv
+/spec/smoke_test/cookies.txt
```

Comment:
Can the `cookies.txt` file be deleted, and then we don't need to add it to the `.gitignore` file?

Reply:
Yeh it actually deleted, but forgot to remove from .gitignore, resolved in 36439d1